Build/Build.cs

```diff
@@ -246,5 +258,22 @@ class Build : NukeBuild
                     (v, path) => v.SetTargetPath(path)));
         });
 
+    Target Spellcheck => _ => _
+        .Executes(() =>
+        {
+            InstallCSpell();
+            RunCSpell();
+        });
+
+    private void InstallCSpell()
+    {
+        Node($"{YarnCli} install --silent", workingDirectory: RootDirectory);
+    }
+
+    private void RunCSpell()
+    {
+        Node($"{YarnCli} run cspell");
```

Comment:
🔧 Node always writes its output to the wrong channel, resulting in "errors" like below. That's why this `LogErrorAsWarning` custom logger was needed in the example. 

![image](https://user-images.githubusercontent.com/572734/210792955-537e67aa-7739-4fa0-bdcd-d8b370fd2942.png)

Reply:
Ahh.. now I get it. Will change that in a second

Reply:
This idea of writing stderr only as warning is quite cool.. learned something for future 👍🏻 Thanks